include/picongpu/CMakeLists.txt

```diff
@@ -293,6 +280,34 @@ elseif(MSVC)
 endif()
 
 
+################################################################################
+# openPMD
+################################################################################
+
+# find openPMD installation
+find_package(openPMD 0.11.0 CONFIG COMPONENTS MPI)
+
+if(openPMD_FOUND)
+    add_definitions(-DENABLE_OPENPMD=1)
+    set(LIBS ${LIBS} openPMD::openPMD)
+endif(openPMD_FOUND)
+
+
+################################################################################
+# ADIOS
+################################################################################
+
+# find adios installation
+#   set(ADIOS_USE_STATIC_LIBS ON) # force static linking
+find_package(ADIOS 1.13.1)
+
+if(ADIOS_FOUND)
```

Comment:
Please add 
```
if(ADIOS_FOUND AND ${ADIOS_VERSION} VERSION_GREATER "2.0.0" )
  set(ADIOS1_FOUND)
```

Please check the cmake file if `ADIOS_FOUND` is used and use `ADIOS1_FOUND` instead to avoid that we will use ADIOS2 for our native ADIOS IO plugin.

If ADIOS2 provides an own find module and a veriable e.g. `ADIOS2_FOUND` then please ignore my comment and drop a note that I can mark this point as solved.

Reply:
ADIOS2 uses its own module, see the usage in the openPMD API [here](https://github.com/openPMD/openPMD-api/blob/878cb83250d435e38492928991f98c65d28b92c3/CMakeLists.txt#L175) and [here](https://github.com/openPMD/openPMD-api/blob/878cb83250d435e38492928991f98c65d28b92c3/CMakeLists.txt#L225).